tools/schemapi/schemapi.py

```diff
@@ -875,8 +987,10 @@ class SchemaBase:
     the _rootschema class attribute) which is used for validation.
     """
 
+    # TODO: Implement `ClassVar` validation using https://peps.python.org/pep-0487/
+
     _schema: ClassVar[dict[str, Any] | Any] = None
-    _rootschema: ClassVar[dict[str, Any] | None] = None
+    _rootschema: ClassVar[dict[str, Any] | Any] = None
```

Comment:
TODO: 

Mockup/implement `ClassVar` validation using Implement `ClassVar` validation using https://peps.python.org/pep-0487/ AKA `__init_subclass__`

Reply:
- Partial Implementation [`06ca339` (#3547)](https://github.com/vega/altair/pull/3547/commits/06ca33929c5f2a5f54d4f15f872844a8c540f605)
- Tests/Demo [`56a43cb` (#3547)](https://github.com/vega/altair/pull/3547/commits/56a43cb244537dc51b28d866e4d6c6b9781ac98c)

Reply:
Since [refactor: Remove `_SchemaBasePEP487` & test suite](https://github.com/vega/altair/pull/3547/commits/9a48448e334c53d2c431bb9a79eb397bc7c0e7d7) I've extracted all of the related code and moved to https://github.com/dangotbanned/altair/tree/schema-base-pep487

Not planning to pursue that any time soon, but thought it would be easier to explain all the outdated comments.